heroku/heroku-pipelines
# add --stage flag completion

---
Thanks for your contribution to heroku-pipelines!
---

## Checklist

The checklist enumerates the tasks you set out to do before the PR becomes ready for review

- [n/a ] Write tests
- [x] Update yarn.lock
- [n/a] Update Dev Center Docs (will be done with autocomplete is GA)
- [n/a] Publish a new entry to [Heroku Changelog](https://devcenter.heroku.com/changelog) (will be done with autocomplete is GA)

## Why
Adding flag options completions for autocomplete

## What are the acceptance criteria for the change?

## How can the change be tested
Tab-complete (in zsh) to see list of pipeline options:

`$ heroku plugins:link heroku-ci`
`$ heroku plugins:update`
`$ heroku ci --pipeline=<TAB>` 

## Demonstration
- If the changeset includes user-visible results, please include before and after screenshots
- If the changeset modifies a workflow, please a screencast/gif illustrating the interaction.

## Who's Affected
All users that have setup heroku-cli autocomplete

## Blockers & upstream dependencies
n/a
